### DISCUSSION TITLE
Can we limit the FPS to a number? Not only VSync

### DISCUSSION THREAD

**Discussion by luojunyuan** [Blinue/Magpie#755] [Ideas]
可不可以指定magpie 缩放窗口的刷新频率一个上限呢，我的显示器是120hz，实际上我认为我只需要30fps就够了，设置垂直同步会稳定120fps，如果不设置垂直同步他会上升至好几百。

我想要降低fps的目的是为了降低功耗，我在使用desktop duplicate模式，如果刷新率fps上限对功耗帮助没有多大影响的话就算了。。

**Comment by Blinue** [comment]
这个功能已经有很多讨论 #378。实际上 #643 已经实现了，经过测试可以有效降低功耗。
